pom.xml

```diff
@@ -83,6 +83,12 @@
     </dependency>
 
     <dependency>
+      <groupId>com.amazon.redshift</groupId>
```

Comment:
I don't think this is in maven central. Amazon's awful about not putting their stuff in maven central.

Probably have to find a mirror or check in the jar with the project.